src/main.ts

```diff
@@ -126,7 +126,7 @@ export class Scraper {
                     }
 
                     // extract todays stars
-                    const todaysStars = domElem.find('.text-gray span:last-child').toArray()[0];
+                    const todaysStars = domElem.find('.text-gray.mt-2 > span.d-inline-block.float-sm-right').toArray()[0];
```

Comment:
I think `.f6.text-gray.mt-2 span:last-child` is better because  The `.d-inline-block.float-sm-right` selector seems to be changed easily. Could you try this?